lib/logstash/filters/syslog_pri.rb

```diff
@@ -104,12 +106,21 @@ def parse_pri(event)
     event.set(@facility_code_key, facility_code)
 
     # Add human-readable names after parsing severity and facility from PRI
-    if @use_labels
-      facility_label = @facility_labels[facility_code]
-      event.set(@facility_label_key, facility_label) if facility_label
-
-      severity_label = @severity_labels[severity_code]
-      event.set(@severity_label_key, severity_label) if severity_label
+    return unless @use_labels
+
+    # from Syslog PRI RFC 4.1.1 PRI Part, facility_code could be at most 124
+    if facility_code > (@facility_labels.size - 1)
```

Comment:
This is awesome! Thanks.

Reply:
Do you think the PR is still good to be merged?